.github/DocsPostProcessor/Sources/DocsPostProcessor/DocsPostProcessor.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+import ArgumentParser
+import SwiftSoup
+
+struct DocsPostProcessor: ParsableCommand {
+    enum Err: Error {
+        case noOverviewFound
+    }
+
+    @Flag(help: "Replaces overview in the left navigation with rendered README")
+    var replaceOverviewWithReadme = false
+
+    @Flag(help: "Replaces readme video link with an embedded vimeo player")
+    var replaceReadmeVideoWithVimeoEmbed = false
+
+    @Argument(help: "The file, or directory with the HTML to change")
+    var path: String
+
+    mutating func run() throws {
+        let url = URL(fileURLWithPath: FileManager.default.currentDirectoryPath).appendingPathComponent(path)
+        var files: [URL] = (url.pathExtension == "html") ? [url] : []
+        if url.pathExtension != "html", let enumerator = FileManager.default.enumerator(at: url, includingPropertiesForKeys: [.isRegularFileKey], options: [.skipsHiddenFiles]) {
+            for case let fileURL as URL in enumerator {
+                let fileAttributes = try fileURL.resourceValues(forKeys:[.isRegularFileKey])
+                if fileAttributes.isRegularFile == true, fileURL.pathExtension == "html" {
+                    files.append(fileURL)
+                }
+            }
+        }
+        for file in files {
+            let html = try String(contentsOf: file, encoding: .utf8)
+            let doc: Document = try SwiftSoup.parse(html)
+            if replaceOverviewWithReadme {
+                try replaceOverviewWithReadme(document: doc)
+                print("Replaced overview with README for: \(file.absoluteString)")
+            }
+            if replaceReadmeVideoWithVimeoEmbed, file.lastPathComponent == "index.html" {
```

Comment:
Not sure what this **should** be, but this causes a failure if you go to protocol docs and then try to go back to the Overview `SwiftCurrent/docs/Protocols/index.html`